CONTRIBUTING.md

```diff
@@ -70,6 +70,11 @@ $ uv pip install -e .
 $ source .venv/bin/activate
 ```
 
+Note that you can modify the existing `.env.dev` dotenv file for running tests via the following:
+```bash
+$ uv run --env-file .env.dev -- python <your-test-script>
```

Comment:
Can we use a realistic example when working with the codebase? Instead of `python <your-test-script>`

Reply:
Updated